### PR TITLE
IA-1880: Upgrading to use caddy and remove the need for pre-created certificates

### PR DESCRIPTION
**Pull request by bushong1**

### CODE REVIEW

File: ecs-task-role.tf [hunk 1]
```diff
@@ -8,21 +8,22 @@ data "aws_iam_policy_document" "ecs_task_principal" {
...
   name               = "fargate-${var.service_name}-task"
   assume_role_policy = data.aws_iam_policy_document.ecs_task_principal.json
 }
-resource "aws_iam_role_policy" "ecs_task" {
-  name   = "fargate-${var.service_name}-task"
-  role   = aws_iam_role.ecs_task.id
-  policy = data.aws_iam_policy_document.ecs_task.json
-}
+
+#data "aws_iam_policy_document" "ecs_task" {
```
Comment (jsmigovsky) [inline ecs-task-role.tf:16]: remove old comments

File: examples/simple/main.tf [hunk 2]
```diff
@@ -0,0 +1,21 @@
+data "aws_vpc" "default" {
+  default = true
+}
+data "aws_subnet_ids" "default" {
+  vpc_id = data.aws_vpc.default.id
+}
+module "simple" {
+  source = "../../"
+  #source  = "USSBA/apex-redirect/aws"
```
Comment (jsmigovsky) [inline examples/simple/main.tf:9]: should these be commented out?

**Review by yaythecloud — COMMENTED** [review]
LGTM

File: ecs-task-role.tf (hunk 1 above)
Comment (bushong1) [reply ecs-task-role.tf:16]: fixed

File: examples/simple/main.tf (hunk 2 above)
Comment (bushong1) [reply examples/simple/main.tf:9]: yeah, the convention in terraform module examples is to just do `../..`, but I like having the direct source/version lines included for ease of copy/paste

**Review by yaythecloud — APPROVED** [review]

**Pull request merged**